src/core/Akka.Remote.TestKit/Conductor.cs

```diff
@@ -74,17 +73,6 @@ public IActorRef Controller
         }
 
         /// <summary>
-        /// Obtain the port to which the controller’s socket is actually bound. This
-        /// will deviate from the configuration in `akka.testconductor.port` in case
-        /// that was given as zero.
-        /// </summary>
-        /// <returns>The address of the controller's socket endpoint</returns>
-        public Task<INode> SockAddr()
```

Comment:
Not since Akka.NET 1.1; it directly exposed old Helios 1.4* members to the public and stopped being used after we switched to Helios 2.0 last year.